ui/src/components/ui/Input.scss

```
@@ -62,5 +63,13 @@
 
 select.field__input {
   cursor: pointer;
-  padding-right: $pad-large;
+  // Draw our own caret instead of the browser default, which macOS crams against the right edge. appearance:none drops the native
```

Stylelint (17.14.0)

[error] Expected empty line before comment (scss/double-slash-comment-empty-line-before)

(scss/double-slash-comment-empty-line-before)